includes/Classifai/Providers/Provider.php

```diff
@@ -171,4 +180,14 @@ public function sanitize_settings( $settings ) {
 		// TODO: Implement sanitize_settings() method.
 		return $settings;
 	}
+
+	/**
+	 * Provides debug information related to the provider.
+	 *
+	 * @return string|array Debug info to display on the Site Health screen. Accepts a string or key-value pairs.
+	 * @since 1.4.0
+	 */
+	public function get_provider_debug_information() {
```

Comment:
@johnwatkins0 as this class is abstract, it can never be instantiated directly so this stub is never called. It may make sense to mark this method as abstract as well to enforce that any providers implement this.